src/types/struct.md

```diff
@@ -29,6 +29,32 @@ A _unit-like struct_ type is like a struct type, except that it has no fields.
 The one value constructed by the associated [struct expression] is the only
 value that inhabits such a type.
 
+## Struct and aggregate values
+
+r[type.struct.value]
```

Comment:
This should say somewhere that a value of strict type is a list of values with one value for each field of the struct.